src/java/com/zimbra/cs/taglib/bean/ZMessageComposeBean.java

```diff
@@ -526,6 +529,43 @@ public void setSendReminderYIM(boolean mSendReminderYIM) {
         this.mSendReminderYIM = mSendReminderYIM;
     }
 
+    public void setColor(String color) {
+        mColor = color;
+    }
+
+    public String getColor() {
+        return mColor;
+    }
+
+    public String getRgbColorValue() {
+        int color = 0;
+        try {
+            color = Integer.parseInt(mColor);
+        } catch (NumberFormatException e) {
+        }
+        return new com.zimbra.common.mailbox.Color((byte)color).toString();
+    }
+
+    public String getColorName() {
+        try {
+            int color = 0;
+            color = Integer.parseInt(mColor);
+            Color colorObj = Color.fromInt(color);
+            return colorObj.getName();
+        } catch (NumberFormatException e) {
```

Comment:
can use more compact `catch (NumberFormatException | ServiceException e)`

Reply:
fixed it. Thank you for your comment.